e2e-tests/playwright/support/pages/rbac.ts

```diff
@@ -37,8 +37,15 @@ export class Roles {
     return [policies];
   }
 
+  //Depending on the version of the Backstage, it can be 'Permission Policies' or 'Accessible Plugins'
+  // Accepts either term
   static getRolesListColumnsText() {
-    return ['Name', 'Users and groups', 'Permission Policies', 'Actions'];
+    return [
+      'Name',
+      'Users and groups',
+      ['Permission Policies', 'Accessible plugins'],
```

Comment:
its not affecting 1.2.x, can we keep only the latest version for 1.3 and avoid these nested columns?

Reply:
@subhashkhileri  It will not break changes for 1.2 since I updated the `verifyColumnHeading`

Reply:
@gustavolira what I mean is we can keep only the latest column name and remove the old one instead of checking either of them.